sirepo/srunit.py

```diff
@@ -90,6 +90,17 @@ def init_auth_db():
     return fc, fc.sr_post('listSimulations', {'simulationType': fc.sr_sim_type})
 
 
+def setup_srdb_root(cfg=None):
+    from pykern import pkunit, pkio
+    import os
+    e = cfg
+    if not e:
+        e = os.environ
+    e.update(
+        SIREPO_SRDB_ROOT=str(pkio.mkdir_parent(pkunit.work_dir().join('db'))),
```

Comment:
Something doesn't smell right. The `db` is definitely not right. That should come from somewhere else in srunit at least.

Reply:
7e0ffcb1b better?

Reply:
7e0ffcb1b better?